styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java

```diff
@@ -736,25 +736,48 @@ void postResource(Resource resource) throws IOException {
     storeWithRetries(() -> runInTransaction(transaction -> {
```

Comment:
Not due to this PR but strange that this method is called `postResource`.

Reply:
This is small enough to get refactored in this PR. So I did it.

Reply:
⭐️